docs/usage.md

```diff
@@ -3,64 +3,63 @@
 - Once record is generated you can apply recommendation generated. [Click Here](#step-to-apply-generated-configuration-recommendations) for step to apply
 
 # Health Assessment
 How to read Health Assessment Record
 - When you navigate to Health Assessment Record's details view. There will be following tabs
-    - **Assessment** : Assessment Tab will contain following things:
+    - **Assessment** : Assessment Tab contains following details:
         - ***System Resources*** : System resources such as vCPUs, RAM, IOPS, and Disk Latency are crucial for determining the overall performance of an application or system.
-            - **vCPUs** represent the processing power allocated to a virtual machine, impacting the system's ability to handle concurrent tasks.
-            - **RAM** is vital for holding active data and instructions, influencing how efficiently applications run.
+            - **vCPUs**
+            - **RAM** 
             - **IOPS (Input/Output Operations Per Second)** measures the speed of disk read/write operations, essential for data-intensive tasks.
-            - **Disk Latency** refers to the time delay in accessing data from storage, with lower latency improving responsiveness.
-        - ***FortiSOAR Details*** :
+            - **Disk Latency** refers to the time delay in accessing data from storage. Lower the latency greater the responsiveness.
+        - ***FortiSOAR Information*** :
             - *Alerts Trend : Number of Alerts Ingested* : This metric provides a comprehensive view of the volume of alerts being ingested into the system, helping to track trends and system load over time.
-            - *Playbook : Most Frequently Executed* : Displays the top ten most executed playbooks, giving insight into which automated processes are most frequently used.
-            - *Playbook : Most Failing* : Lists the top ten playbooks that fail most often, providing a focus area for troubleshooting and improvement.
-            - *Playbook : Running In Debug Mode* : Identifies the top ten playbooks currently running in debug mode, typically for development or issue resolution purposes.
-            - *Playbook : Most Time Consuming* : Shows the top ten playbooks consuming the most time, helping to identify performance bottlenecks or areas for optimization.
-    - **Recommendation** : Recommendation Tab will contain following things
-        - *PostgreSQL* : Adjust database settings for improved performance.
-        - *PHP-FPM* : Adjust php-fpm settings to boost processing speed.
+            - *Playbooks : Most Frequently Executed* : Displays the top ten most executed playbooks, giving insight into which automated processes are frequently used.
+            - *Playbooks : Most Failing* : Lists the top ten playbooks those fail often, providing a focus area for troubleshooting and improvement.
+            - *Playbooks : Running In Debug Mode* : Identifies the top ten playbooks currently running in debug mode.
```

Comment:
for playbook mode detailed info, you can add prod doc link here with ref to pb mode chapter